pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.black]
```

Comment:
I was wondering if you had thoughts on using .toml for package management too, with something like [poetry](https://python-poetry.org/) instead of the .cfg file?

Reply:
i am definitely planning to move to a `pyproject.toml`-only package setup+config, this was just a smaller first step. iirc i couldn't go directly there without upgrading python (or maybe just setuptools?) further than i wanted to go at this point. as a matter of personal preference, i don't really like poetry -- though i do respect it -- but modern setuptools w/ pip, build, etc. are quite capable these days.